code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -461,7 +461,7 @@
 	return output_text
 
 /mob/living/carbon/human/assembleHeightString(mob/examiner)
-	var/heightString = null
+	var/list/heightString = null
```

Comment:
This list needs to be properly initialised by doing `= list()`

Reply:
I think I've done that